examples/8_benchmark_parallel.cpp

```diff
@@ -148,10 +148,14 @@ int main(
     // * The number of iterations. Effectively, the benchmark will be repeated for this many
     //   times.
 #if openPMD_HAVE_ADIOS1 || openPMD_HAVE_ADIOS2
-    benchmark.addConfiguration("bp", dt, 10);
+    benchmark.addConfiguration(
+        R"({"adios2": {"dataset":{"operators":[{"type": "blosc"}]}}})",
```

Comment:
Just to be sure, you mean zstd *instead* of blosc? Or is there a way to pick zstd inside blosc?

Reply:
> Just to be sure, you mean zstd instead of blosc? Or is there a way to pick zstd inside blosc?

The latter! blosc is a meta-compressor and implements threaded compression routines for zlib, zstd, lz4, blosclz, etc.:
https://www.blosc.org/pages/blosc-in-depth